src/templates/doe_workflow_attic_insulation.mdx

```diff
@@ -0,0 +1,188 @@
+        and when it is needed, the strategy is to seal air leaks, especially high (attic) and low (basement
+        or crawlspace) and then provide continuous mechanical ventilation with optional filtering or
+        other conditioning.
+        
+        ## Insulation Details
+
+        <Photo id="insulation_measuring_rulers_photo" label="Insulation Measuring Rulers  – Photo" required>
+            Overview of insulation measuring rulers, one installed every 300 sq. ft. visible 
+            from attic entrance (photo taken from attic entrance).     
+        </Photo>
+
+        <Photo id="rvalue_table_photo" label="R-value Per Inch  – Photo" required>
+        Photo of insulation bag R-value per inch table    
+        </Photo>
+
+        <Photo id="skylight_shaft_photo" label="Skylight Shaft – Photo">
+        Photo of skylight shaft insulation detail
+        </Photo>
+
+        <Photo id="knee_wall_photo" label="Knee Wall – Photo">
+            Photo of knee wall insulation detail     
+        </Photo>
+
+        <Photo id="stairwell_wall_photo" label="Stairwell Wall – Photo">
+        Photo of stairwell wall insulation detail     
+        </Photo>
+
+        <Photo id="storage_platform_photo" label="Storage Platform – Photo">
+        Photo of storage platform insulation detail    
+        </Photo>
+
+        <Photo id="wide_angle_photo" label="Wide Angle – Photo" required>
+        Wide angle photo showing insulation uniformity with rulers visible
+        </Photo>
+
+        <Photo id="attic_door_hatch_photo" label="Attic Door/Hatch - Photo" required>
+        Photo of insulation on attic door/hatch
+        </Photo>
+        
+        ## Post Air Sealing And Insulation Assessment
+
+        <Photo id="blowerdoor_setup_photo" label="Blower Door Setup - Photo" required>
+            Photo of blower door setup
+        </Photo>
+    
+        <Photo id="blowerdoor_cfm50_photo" label="Manometer CFM50 - Photo" required>
+            Photo of blower door manometer showing CFM at 50 Pa read out
+        </Photo>
+
+        | Leakage Reduction Calculator                                                                                                                                                                                                                                           |
+        | ----------------------------
+        | Final CFM50 = { props.data.ACH50_calculator?.final_cfm50 ? props.data.ACH50_calculator?.final_cfm50 : null } 
+        | Initial CFM50 = { props.data.ACH50_calculator?.ini_cfm50 ? props.data.ACH50_calculator?.ini_cfm50 : null }  
+        | Building Volume (ft&#179;) = { props.data.ACH50_calculator?.volume ? props.data.ACH50_calculator?.volume : null } 
+        | Initial ACH50 = { props.data.ACH50_calculator?.ini_cfm50 && props.data.ACH50_calculator?.volume ? ((60 * props.data.ACH50_calculator.ini_cfm50) / props.data.ACH50_calculator.volume).toFixed(2) : null } 
```

Comment:
let's write out `.initial_cfm50` it is only three more characters.

Reply:
done